molecule/additional_flags/converge.yml

```diff
@@ -10,6 +10,7 @@
         name: ansible-role-clamav
       vars:
         clamav_cron_frequency: daily
+        clamav_scan_start_timer: true
```

Comment:
Please move this to maintain alphabetical ordering of the variables.

Reply:
Please see commit 711d0c0.